utensor_cgen/backend/utensor/code_generator/rearch/_code_generator.py

```diff
@@ -83,33 +91,60 @@ def _find_required_ops(self, ugraph):
         )
     return ops, placeholders, tensor_var_map
 
-  def _try_get_opt_snippets(self, ugraph, ops, tensor_var_map):
+  def _try_get_opt_snippets(self, ugraph, ops, tensor_var_map, placeholders):
```

Comment:
`_try_get_op_snippets`?

Reply:
Well, the order of the snippets returned is optimized for memory usage (by applying tensor life cycle analysis).
If you have better name, I'm open for any suggestion.

Reply:
oh no, was just suggesting we get rid of the t after op :)

Reply:
`opt` means optimized XD